### PR TITLE
feat: Deployment options. Use pre-built package by default.

### PR DESCRIPTION
**Pull request by baolsen**

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# TEST REQUIREMENTS
+
+# To use: pip install -r tests/requirements.txt
+
+# Include main requirements
+-r ../costnotifier/requirements.txt
+
+# Additional packages to install when doing testing
```
Comment (baolsen) [inline tests/requirements.txt:8]: One nice thing with this change.
Only install boto3 when doing tests. It is already available within the lambda environment :D

File: main.tf [hunk 2]
```diff
@@ -36,16 +57,21 @@ module "billing_notifier_lambda" {
   function_name = var.naming_prefix
   description   = var.lambda_description
 
-  handler = "handler.report_cost"
+  handler = "app.lambda_handler"
   runtime = var.runtime
   timeout = 300
```
Comment (baolsen) [inline main.tf:63]: Removing this means we no longer depend on the remote build environment

**Comment by baolsen** [comment]
I'll also do some testing in TF in our internal environments to check the lambda is still working :) 
Will report back.

**Review by grasshopper-josh — APPROVED** [review]

**Comment by baolsen** [comment]
Tagged with 0.42.0 for testing and rolled to our root account

**Comment by baolsen** [comment]
Tests passed, merging.

**Pull request merged**